lib/spack/spack/build_systems/python.py

```diff
@@ -211,7 +211,9 @@ def install(self, spec, prefix):
     def install_args(self, spec, prefix):
         """Arguments to pass to install."""
         args = ['--prefix={0}'.format(prefix)]
-
+        args += ['--install-lib={0}'.format(os.path.join(prefix, 'lib',
+                                            "python{0}".format(self.spec['python'].version.up_to(2)),
+                                            'site-packages'))]
```

Comment:
That line looks very long. Upstream's PRs normally need to pass some flake8 runs, it may be better to break this a few more times (and only assign to `args` once)